### ISSUE TITLE
Auto reloading all the time

### ISSUE THREAD

**Issue by NikitaKazancev** [JetpackDuba/Gitnuro#251]
Hello, in version 1.3.0 there was an improvement "Restored Git directory changes detection".

The problem with this feature happens when you work with quite a large repo. Namely: I have such "auto reloading" like all the time (it happens like every 10 seconds so I can't just use the app).

Right now I returned to 1.2.1 version because of that. 
Could you please add a small flag/toggle into settings so we (users) could turn off auto-reload if we don't need it.

**Comment by JetpackDuba** [comment]
Hello,

I'm guessing you have the same issue in the latest release, right? I did some changes regarding detection but not sure if that solves your issues.

Is the repo's content changing constantly? Or it triggers without any changes?

In any case, v1.4.1 had a bug that disabled completely auto reload, if you would prefer using that.

**Comment by JetpackDuba** [comment]
True, there is a sneaky bug there! I didn't notice it before because, as you mentioned, it is only noticeable in large repos.

I may create a release 1.4.3 with that fix. I'll be creating some tests to prevent regressions from happening again. Thanks!
